### PR TITLE
switch to gradle

### PR DESCRIPTION
**Pull request by loganj**
**REVIEW ONLY - DO NOT MERGE**

Java source changes conflict with other outstanding branches.  Will
rebase and correct after those PRs land.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      mavenCentral()
+    }
+  }
+
+  repositories {
+    mavenCentral()
+  }
+
+  group = GROUP
+  version = VERSION_NAME
```
Comment (rjrjr) [inline build.gradle]: GROUP? VERSION_NAME?

Comment (loganj) [inline build.gradle]: from gradle.properties.

Comment (rjrjr) [inline build.gradle]: NM, found it.

**Comment by loganj** [comment]
Rebased, bumped gradle plugin to 1.1

**Comment by rjrjr** [comment]
LGTM

**Comment by loganj** [comment]
Rebased again.

**Pull request merged**